core/src/player.rs

```diff
@@ -1421,6 +1421,7 @@ impl Player {
         })
     }
 
+    #[instrument(level = "debug", skip_all)]
```

Comment:
could you also instrument `tick()`? It would mark the the "most uninterrupted non-render cpu time spend in a row"

Reply:
Sure thing!